variables.tf

```diff
@@ -130,9 +130,13 @@ variable "databases" {
 }
 
 variable "redis_version" {
```

Comment:
maybe it will be better to rename it to valkey in the next major version of module

Reply:
in the next major version of module